.github/workflows/publish.yaml

```diff
@@ -7,16 +7,19 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
```

Comment:
These permissions are noted in https://github.com/JasonEtco/build-and-tag-action/pull/45 and match a similar workflow that's found in [`slackapi/slack-github-action`](https://github.com/slackapi/slack-github-action/blob/47b1876080be991b1fc5ff91d384d0042e769e4e/.github/workflows/publish.yml#L11-L12)